charts/workflows/charts/move2kube/templates/01-sonataflow_m2k.yaml

```diff
@@ -2,15 +2,14 @@ apiVersion: sonataflow.org/v1alpha08
 kind: SonataFlow
 metadata:
   annotations:
-    sonataflow.org/description: Workflow to execute Move2Kube
+    sonataflow.org/description: Workflow to execute {{ .Values.workflow.name }}
```

Comment:
we can't support that becuase this file auto-generated by the CI of serverless-workflows.

Reply:
@gabriel-farache  I suggest to remove all the parameterization inside the manifests for now,  and to leave the changes that  solve the problem of $subject first.

Reply:
I'll split in commits and open another PR just to keep those changes somewhere in case we need it and/or as a reminder to what I'd like to have parametrized